379 % cvs diff -p as/{notes,as.h,dwarf2dbg.h,arm.c} otool/{notes,print_objc2_32bit.c,print_objc2_64bit.c}
Index: as/as.h
===================================================================
RCS file: /cvs/root/cctools/as/as.h,v
retrieving revision 1.10
diff -p -r1.10 as.h
*** as/as.h	2 Jul 2010 01:14:33 -0000	1.10
--- as/as.h	28 Mar 2012 23:38:13 -0000
*************** extern cpu_subtype_t archflag_cpusubtype
*** 179,185 ****
  extern char *specific_archflag;
  
  /* TRUE if the .subsections_via_symbols directive was seen */
! int subsections_via_symbols;
  
  /* -I path options for .includes */
  struct directory_stack {
--- 179,185 ----
  extern char *specific_archflag;
  
  /* TRUE if the .subsections_via_symbols directive was seen */
! extern int subsections_via_symbols;
  
  /* -I path options for .includes */
  struct directory_stack {
Index: as/dwarf2dbg.h
===================================================================
RCS file: /cvs/root/cctools/as/dwarf2dbg.h,v
retrieving revision 1.3
diff -p -r1.3 dwarf2dbg.h
*** as/dwarf2dbg.h	2 Jul 2010 01:14:33 -0000	1.3
--- as/dwarf2dbg.h	28 Mar 2012 23:38:13 -0000
*************** extern void dwarf2_emit_label (symbolS *
*** 115,121 ****
  /* True when we're supposed to set the basic block mark whenever a label
     is seen.  Unless the target is doing Something Weird, just call 
     dwarf2_emit_label.  */
! bfd_boolean dwarf2_loc_mark_labels;
  
  extern void dwarf2_finish (void);
  
--- 115,121 ----
  /* True when we're supposed to set the basic block mark whenever a label
     is seen.  Unless the target is doing Something Weird, just call 
     dwarf2_emit_label.  */
! extern bfd_boolean dwarf2_loc_mark_labels;
  
  extern void dwarf2_finish (void);
  
Index: as/arm.c
===================================================================
RCS file: /cvs/root/cctools/as/arm.c,v
retrieving revision 1.83
diff -p -r1.83 arm.c
*** as/arm.c	15 Mar 2012 21:29:46 -0000	1.83
--- as/arm.c	28 Mar 2012 23:38:13 -0000
*************** typedef enum {
*** 151,158 ****
  # define N_(String) (String)
  
  /* STUFF FROM gas/as.h */
! #define COMMON
! COMMON subsegT now_subseg;
  
  /* STUFF FROM gas/config/tc-arm.h */
  #define ARM_FLAG_THUMB 		(1 << 0)	/* The symbol is a Thumb symbol rather than an Arm symbol.  */
--- 151,157 ----
  # define N_(String) (String)
  
  /* STUFF FROM gas/as.h */
! extern subsegT now_subseg;
  
  /* STUFF FROM gas/config/tc-arm.h */
  #define ARM_FLAG_THUMB 		(1 << 0)	/* The symbol is a Thumb symbol rather than an Arm symbol.  */
Index: otool/print_objc2_32bit.c
===================================================================
RCS file: /cvs/root/cctools/otool/print_objc2_32bit.c,v
retrieving revision 1.7
diff -p -r1.7 print_objc2_32bit.c
*** otool/print_objc2_32bit.c	31 Jul 2009 00:57:06 -0000	1.7
--- otool/print_objc2_32bit.c	28 Mar 2012 23:38:13 -0000
*************** struct category_t {
*** 259,265 ****
      uint32_t protocols;		/* struct protocol_list_t * (32-bit pointer) */
      uint32_t instanceProperties; /* struct objc_property_list *
  				    (32-bit pointer) */
! } category_t;
  
  static
  void
--- 259,265 ----
      uint32_t protocols;		/* struct protocol_list_t * (32-bit pointer) */
      uint32_t instanceProperties; /* struct objc_property_list *
  				    (32-bit pointer) */
! };
  
  static
  void
Index: otool/print_objc2_64bit.c
===================================================================
RCS file: /cvs/root/cctools/otool/print_objc2_64bit.c,v
retrieving revision 1.9
diff -p -r1.9 print_objc2_64bit.c
*** otool/print_objc2_64bit.c	31 Jul 2009 00:57:06 -0000	1.9
--- otool/print_objc2_64bit.c	28 Mar 2012 23:38:13 -0000
*************** struct category_t {
*** 262,268 ****
      uint64_t protocols;		/* struct protocol_list_t * (64-bit pointer) */
      uint64_t instanceProperties; /* struct objc_property_list *
  				    (64-bit pointer) */
! } category_t;
  
  static
  void
--- 262,268 ----
      uint64_t protocols;		/* struct protocol_list_t * (64-bit pointer) */
      uint64_t instanceProperties; /* struct objc_property_list *
  				    (64-bit pointer) */
! };
  
  static
  void
